api/v2/controllers/commentController.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable radix */
+import articles from '../models/article';
+import validation from '../middleware/validation';
+
+class ArticleComment {
+  // comment on a posted article
+  static async commentArticle(req, res) {
+    const article = articles.find((a) => a.id === parseInt(req.params.id));
+    if (!article) {
+      return res.status(404).json({
+        status: 404,
+        error: 'article not found',
+      });
+    }
+
+    const { error } = validation.validateComment(req.body);
```

Comment:
A method should only do what it was created for, in this case, it was created to create a comment on an article and nothing else, so you should move the validation in its separate folder and apply it in the route as a middleware.